packages/sit-onyx/src/components/OnyxTag/OnyxTag.stories.ts

```diff
@@ -46,3 +50,29 @@ export const WithTruncation = {
     style: "max-width: 10rem",
   },
 } satisfies Story;
+
+/**
+ * This example shows the interactive tag.
+ */
+export const Interactive = {
+  argTypes: {
+    ...withNativeEventLogging(["onClick"]),
+  },
+  args: {
+    label: "Tag",
+    clickable: "clickable",
```

Comment:
Make sure to use good examples for tooltips to encourage helpful texts:

```suggestion
    clickable: "Click to add 'Tag' to the selection.",
```